recipes/clickhouse-cpp/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Thank you for adding test_v1_package. You can ignore it in the future, as it's no longer required for new recipes: https://github.com/conan-io/conan-center-index/discussions/17406

Reply:
Thank you, I missed that we can ignore it now